doc/rst/language/spec/statements.rst

```diff
@@ -810,21 +810,23 @@ module-level symbols would be inserted into this enclosing scope with
 the same name, and that name is referenced by other statements in the
 same scope as the use.
 
-Use statements are transitive by default: if a module A uses a module B,
-and module B contains a use of a module or enumerated type C, then C’s
-public symbols may also be visible within A. The exception to this
-occurs when B has public symbols that shadow symbols with the same name
-in C, or when the use of C has been declared explicitly ``private``. If
-a use statement is declared to be ``private``, then the symbols it makes
-visible will only be visible to the scope containing the use.
-
-This notion of transitivity extends to the case in which a scope imports
-symbols from multiple modules or constants from multiple enumeration
-types. For example if a module A uses modules B1, B2, B3 and modules B1,
-B2, B3 use modules C1, C2, C3 respectively, then all of the public
-symbols in B1, B2, B3 have the potential to shadow the public symbols of
-C1, C2, and C3. However an error is signaled if C1, C2, C3 have public
-module level definitions of the same symbol.
+Use statements may be explicitly delared ``public`` or ``private``.
+By default, uses are ``private``.  Making a use ``public`` causes its
+symbols to be transitively visible: if module A uses module B, and
+module B contains a public use of a module or enumerated type C, then
+C’s public symbols will also be visible to A unless they are shadowed
+by symbols of the same name in B.  Conversely, if B's use of C is
+``private`` then A will not be able to see C's symbols by virtue of
```

Comment:
Maybe "as a result of" instead of "by virtue of"?

Reply:
How about "due to"?

Reply:
That works!